testsuite/unittests/test_gap_junction.sli

```diff
@@ -82,7 +98,7 @@ neuron_no_gap neuron_no_gap Connect
 0 /max_delay get /old_max Set
 0 /min_delay get /old_min Set
 
-neuron_gap neuron_gap 4.0 4.0 /gap_junction Connect
+neuron_no_gap neuron_no_gap 2.0 2.0 Connect
```

Comment:
@heplesser Please see the improved version of the test. The original idea was to test, that gap-junction connections contribute to the calculation of the min- and max-delay with the value of `wfr_comm_interval`. Therefore I added two test cases in the new version:

In the first version the min- and max-delay is set to 3.0 ms and then a gap-junction connection is added. It is tested that the min-delay changes to 2.0 ms (the value of `wfr_comm_interval`) and that the max-delay stays unchanged.

In the second version the min- and max-delay is set to 1.0 ms and it is tested that the max-delay changes to 2.0 ms after the creation of the gap-junction connection and that the min-delay stays unchanged.